src/routes/+layout.server.ts

```diff
@@ -3,7 +3,8 @@ import { fetchUserData } from '$lib/fetchRequests';
 import { handleAuthenticatedRequest } from '$lib/requestHandler';
 import { get_user_role } from '$lib/getrole';
 
-export const load: LayoutServerLoad = async ({ cookies }) => {
+export const load: LayoutServerLoad = async (event) => {
+	const { cookies } = event;
```

Comment:
skulle mene at layoutServerLoad har en parameter der hedder cookies, og er måske ikke nødvendigt med destructoring af event for at få cookies

Reply:
nej